tests/pathfinding/test_claim_fees.py

```diff
@@ -96,5 +97,7 @@ def mock_connect_to_blockchain(monkeypatch):
 @pytest.mark.usefixtures('mock_connect_to_blockchain')
 def test_cli(default_cli_args):
     runner = CliRunner()
-    result = runner.invoke(main, default_cli_args, catch_exceptions=False)
+    result = runner.invoke(
+        main, default_cli_args + ['--state-db=:memory:'], catch_exceptions=False
```

Comment:
Can we put `--state-db=:memory:` into `default_cli_args`

Reply:
Yes, actually we already did. There are just two different `default_cli_args`. I'll make a shared one.